forge-game/src/main/java/forge/game/GameAction.java

```diff
@@ -1462,7 +1466,7 @@ else if (c.hasBeenDealtDeathtouchDamage() || (c.getDamage() > 0 && c.getLethal()
         }
 
         if (game.getAge() != GameStage.Play) {
-            return;
+            return performedSBA;
```

Comment:
Should this just return false if our GameStage isn't Play?

Reply:
Most likely it will not provide any advantage